README.md

```diff
@@ -1,5 +1,10 @@
 # Keras Applications
 
+:warning: **This repository is deprecated**: Keras applications are still
+supported, but all application code now lives in the main keras repository,
```

Comment:
This model is not included in the main repo. https://github.com/keras-team/keras-applications/blob/bc89834ed36935ab4a4994446e34ff81c0d8e1b7/keras_applications/resnext.py#L24